docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md

```diff
@@ -624,9 +709,36 @@ Documentation in this section is preliminary and subject to change.
 
 ### Decode transaction events
 
-:::note
-Documentation in this section is preliminary and subject to change.
-:::
+You might be interested into decoding events emitted by a contract. You can do so by using the [`TransactionEventsParser`](#).
+
+Suppose we'd like to decode a `startPerformAction` event emitted by the [multisig](https://github.com/multiversx/mx-contracts-rs/tree/main/contracts/multisig) contract.
+
+Let's fetch an already processed [transaction](https://devnet-explorer.multiversx.com/transactions/05d445cdd145ecb20374844dcc67f0b1e370b9aa28a47492402bc1a150c2bab4), to serve as an example, and convert it to a [TransactionOutcome](https://multiversx.github.io/mx-sdk-py/_modules/multiversx_sdk/core/transactions_outcome_parsers/resources.html#TransactionOutcome).
+
+```py
+from multiversx_sdk import ApiNetworkProvider, TransactionsConverter
```

Comment:
Still with the legacy converter :see_no_evil: 

All right, we'll get rid of it in v1.